tests/components/todoist/conftest.py

```diff
@@ -70,6 +70,7 @@ def make_api_task(
         section_id=None,
         url="https://todoist.com",
         sync_id=None,
+        duration=None,
```

Comment:
Both `duration` and `can_assign_tasks` were added in 2.1.4, updating test fixtures to account for them. https://github.com/Doist/todoist-api-python/compare/v2.1.3...v2.1.4#diff-ca4b2433cbd264e0ee94ac8cab43180ae7e28ca4460f12c73d983275cbc8d676